src/main/java/justin/Ui.java

```diff
@@ -0,0 +1,153 @@
+package justin;
+
+import justin.task.Task;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a class that is responsible for
+ * sending outputs to users.
+ * @author Justin Cheng.
+ */
+public class Ui {
+    private final static String DIVIDER = "--------------------------------------------------------";
+
+    private final static String SEPARATOR = System.lineSeparator();
+
+    private final static String WELCOME_MESSAGE = "Hi, I'm Justin. What do you want me to do for you?";
+    private final static String GOODBYE_MESSAGE = "Bye! Hope to see you again soon.";
+    private final static String UNMARK_MESSAGE = "OK, I've marked your task as undone: ";
+    private final static String MARK_MESSAGE =  "Nice! I have marked this task as done: ";
+    private final static String LIST_MESSAGE = "Here are the tasks in your list: ";
+    private final static String DELETE_MESSAGE = "OK, I have removed the following task from the list: ";
+    private final static String ADD_MESSAGE = "Got it, I have added the following into the list: ";
+    private final static String FIND_MESSAGE = "Here are the matching tasks in your list: ";
```

Comment:
Nit: could these messages possibly be moved to each corresponding command class?

Reply:
Hmmm just a thought about this, would it be better to have all the messages in the Ui class since the class is supposed to deal with what's shown to the user? At the same time, I also feel putting each in their individual command class is neater and more direct?

Reply:
Proper declaration of constants.